README.md

```diff
@@ -2208,25 +2217,32 @@ Other Style Guides
     > Why? It provides precision by distinguishing null/undefined from other falsy values, enhancing code clarity and predictability.
 
     ```javascript
-    // bad
-    const value = 0 ?? 'default';
-    // returns 0, not 'default'
-
-    // bad
-    const value = '' ?? 'default';
-    // returns '', not 'default'
```

Comment:
?? operator is not bad! It solves issues with || operator. 
That's why we should show it in brighter light

Reply:
it's bad when you care about falsiness.